### PR TITLE
Sprint7 103 wvr button

### LINKED ISSUE #103: Wvr Components should provide a `wvr-button` component
The implementation should afford all the design options exposed by the default bootstrap 4 buttons:

https://getbootstrap.com/docs/4.0/components/buttons/

And to the extent that it is necessary utilize the affordances of ngBootstraps buttons:

https://ng-bootstrap.github.io/#/components/buttons/examples

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #103

### CODE REVIEW

File: projects/wvr-elements/src/lib/wvr-button/wvr-button.component.ts [hunk 1]
```diff
@@ -209,15 +209,15 @@ export class WvrButtonComponent extends WvrAbstractBaseComponent {
...
 
   /**
    * The weaver footer component constructor
    * @param elementRef: ElementRef - a reference to the button element.
    * @param cdRef: ChangeDetectorRef - utilized for change detection.
    */
-  constructor(private readonly cdRef: ChangeDetectorRef, private readonly elementRef: ElementRef) {
-    super();
-  }
+  // constructor(private readonly cdRef: ChangeDetectorRef, private readonly elementRef: ElementRef) {
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-button/wvr-button.component.ts:219]: this should not be commented out

File: projects/wvr-elements/src/lib/wvr-button/wvr-button.component.ud-examples.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  <wvr-button btn-class="outline-light" wvr-btn-type="button">
+    <wvr-text value="Outline Light "></wvr-text>
+  </wvr-button>
+  <wvr-button btn-class="outline-dark" wvr-btn-type="button">
+    <wvr-text value="Outline Dark "></wvr-text>
+  </wvr-button>
+  </snippet>
+</example>
+
+<example name="Weaver Button Styled">
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-button/wvr-button.component.ud-examples.html:93]: Please change to "sized"

File: src/index.html [hunk 3]
```diff
@@ -274,59 +274,39 @@
   </div>
 
   <br /><hr /><br />
-  <div class="container" >
-    <!-- style="text-align: center;" -->
+  <div class="container" style="text-align: center;">
     <!-- border-color="1px solid var(--wvr-black)" -->
```
Comment (jeremythuff) [inline src/index.html:278]: This comment is not needed

File: projects/wvr-elements/src/lib/wvr-button/wvr-button.component.ts (hunk 1 above)
Comment (rmathew1011) [reply projects/wvr-elements/src/lib/wvr-button/wvr-button.component.ts:219]: Resolved

File: src/index.html (hunk 3 above)
Comment (rmathew1011) [reply src/index.html:278]: Removed comment

**Pull request merged**